CONTRIBUTING.rst

```diff
@@ -246,21 +241,44 @@ Stylistic Guidelines
 * For Cython functions, release the GIL whenever possible, using
   ``with nogil:``.
 
-
 Testing
 -------
 
-See the testing section of the Installation guide.
+The test suite must pass before a pull request can be merged, and
+tests should be added to cover all modifications in behavior.
+
+We use the `pytest <https://docs.pytest.org/en/latest/>`__ testing
+framework, with tests located in the various
+``skimage/submodule/tests`` folders.
+
+Testing requirements are listed in `requirements/test.txt`.
+Run:
+
+- **All tests**: ``spin test``
+- Tests for a **submodule**: ``spin test skimage/morphology``
+- Run tests from a **specific file**: ``spin test skimage/morphology/tests/test_gray.py``
+- Run **a test inside a file**:
+  ``spin test skimage/morphology/tests/test_gray.py::test_3d_fallback_black_tophat``
```

Comment:
```suggestion
  ``spin test -- skimage/morphology/tests/test_gray.py::test_3d_fallback_black_tophat``
```

Reply:
This works, as-is, but also fine to change it.

Reply:
Yeah, not a sure if including this is more confusing or less. :smile: